### PR TITLE
Include takeshia recieving as correct answer

### PR DESCRIPTION
**Pull request by rnice01**
Hello, I'm _pretty_ sure this answer is also correct as it marks たけしさん as the subject and the verb is もらう.

### CODE REVIEW

**Comment by SethClydesdale** [comment]
Hello @rnice01! Thank you for taking interest in contributing.

In regards to this answer, the reason it is incorrect has to do with the following footnote from Genki II (pg.57):
> もらう is like くれる and implies that you identify yourself more closely with the recipient than with the giver. Thus it is wrong to use もらう if *you* receive from *me*, for example. (It is one indication that nobody can be detached from their ego.)
> ×（あなたは）私から手紙をもらいましたか。　　*Did you receive a letter from me?*

Someone else has actually asked a question about this before (https://github.com/SethClydesdale/genki-study-resources/issues/35). I think it's pretty common to overlook this, because Genki's explanation is a bit convoluted. I'll quote the answer I gave before which should help clarify what was explained.

> Genki's explanation is a little convoluted, but basically もらう is seen from the receiver's point of view, so when it comes to the first person, _others_ usually don't receive things from _you_.

Basically, the giver -- which is you in this exchange -- cannot use もらう to describe the exchange.

I'm hoping that they improve the explanation of this construct in the 3rd edition, because even I had trouble figuring it out at first.

If any questions, let me know!

**Comment by rnice01** [comment]
@SethClydesdale oh that's slightly embarassing :laughing: but thanks for the great explanation!

**Comment by SethClydesdale** [comment]
No worries, and you're welcome! 😉
